src/components/ebay-chip/index.marko

```diff
@@ -0,0 +1,19 @@
+$ const textId = component.elId("text");
```

Comment:
So if we don't allow the user to pass in a custom textid we should not do it this way but rather use `:scoped`
https://markojs.com/docs/class-components/#scoped

Reply:
This is nicer, thanks